lib/auth/touchid/credentials.m

```diff
@@ -113,3 +117,88 @@ int FindCredentials(LabelFilter filter, CredentialInfo **infosOut) {
   CFRelease(items);
   return infosLen;
 }
+
+int FindCredentials(LabelFilter filter, CredentialInfo **infosOut) {
+  return findCredentials(YES /* applyFilter */, filter, infosOut);
+}
+
+int ListCredentials(const char *reason, CredentialInfo **infosOut,
+                    char **errOut) {
+  LAContext *ctx = [[LAContext alloc] init];
+
+  __block LabelFilter filter;
```

Comment:
What is `__block`?

Reply:
Approved, so it must to be right? 😅 
https://stackoverflow.com/a/7081005

Reply:
Jakub beat me to it, but it's so we can read/write to it in the ObjC block below. ([The __block Storage Type](https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/Blocks/Articles/bxVariables.html#//apple_ref/doc/uid/TP40007502-CH6-SW6).)